src/Symfony/Component/Form/Tests/Extension/Core/DataMapper/PropertyPathMapperTest.php

```diff
@@ -357,4 +357,28 @@ public function testMapFormsToDataIgnoresDisabled()
 
         $this->mapper->mapFormsToData(array($form), $car);
     }
+
+    public function testMapFormsToDataDoesNotChangeEqualDateTimeInstance()
```

Comment:
This test would have passed before, right? We should probably add one making use of `DateTimeImmutable`.

Reply:
Done, but this can be simplified back in 3.4.